spec/unit/module_repository/forge_spec.rb

```diff
@@ -19,15 +34,33 @@
     expect(forge.forge).to eq 'forge.example.local'
   end
 
-  describe "and the expected version is :latest", :vcr => true do
+  describe "and the expected version is :latest" do
     subject(:forge) { described_class.new }
 
     before do
-      forge.conn.builder.insert_before(Faraday::Adapter::NetHttp, VCR::Middleware::Faraday)
+      boolean_body = <<-JSON
+      {"releases": [
+          {"version": "1.0.1"},
+          {"version": "1.0.0"},
+          {"version": "0.9.0"}
+        ]
+      }
+      JSON
+
+      spotty_body = <<-JSON
+      {"releases": [
+          {"version": "0.4.0", "deleted_at": "some point"},
+          {"version": "0.3.0"}
+        ]
+      }
+      JSON
```

Comment:
The original intent was to use the same faraday middleware as the actual application which would parse the JSON and ignore any whitespace, but that was causing issues with Ruby 1.8.7 so I replaced this with the actual data structures.